chromebook-setup.sh

```diff
@@ -665,7 +665,7 @@ cmd_build_vboot()
             ;;
     esac
 
-    echo "root=PARTUUID=%U/PARTNROFF=1 rootwait rw ${extra_kparams}" > boot_params
+    echo "root=/dev/sda2 rootwait rw ${extra_kparams}" > boot_params
```

Comment:
This change will break some installations.. Hard-coding to '/dev/sda2' doesn't look safe to me, while it might work on most cases if you boot from USB it will not work if you boot from a MMC or eMMC.

Why you can't use the method used before?

Reply:
Ups, that was a local patch that I pushed it by mistake. The reason why I had that in my tests is that `dracut` seems to not be happy with `PARTUUID=%U/PARTNROFF=1`. I remember we discussed this at the time but never proposed a fix for `dracut`?

Anyways, my plan was to propose after this PR lands a `root` option for `chromebook-setup.sh` so that users could override the default `PARTUUID=%U/PARTNROFF=1`. But in the meantime just hardcoded to `/dev/sda2` in my test setup. Sorry for missing that pushed this commit too.